### ISSUE TITLE
Easier deployment

### ISSUE THREAD

**Issue by zmoon** [fonsp/pluto-on-binder#3]
Wondering if this could be added to PyPI or jupyter-server-proxy's [contrib](https://github.com/jupyterhub/jupyter-server-proxy/tree/master/contrib) to make it easy for people to create their own Pluto Binder setup with custom Julia Manifest or other custom Binder settings.

**Comment by fonsp** [comment]
Yeah! What we're looking into right now is to put everything (including the python stuff) into a single `Dockerfile`, and you would make you repository/gist pluto-binderable just by adding that Dockerfile, and optionally a Manifest.toml

**Comment by fonsp** [comment]
But we might want to register it anyway for https://github.com/fonsp/PlutoUtils.jl/issues/4

**Comment by AdrianSkib** [comment]
Hi, has there been any progress on this? I love what you've done here! It would be great to set up my own binder server, as I'm trying to share a quite computationally heavy notebook.

**Comment by fonsp** [comment]
Hey @AdrianSkib ! No progress here yet!

I can't promise that I will be able to help you (the binder stuff can be a bit time consuming...), but can you share some info about your situation?
